### ISSUE TITLE
Add automated GH release action

### ISSUE THREAD

**Issue by winsmith** [TelemetryDeck/SwiftSDK#159]
Similar to https://github.com/TelemetryDeck/FlutterSDK/blob/main/.github/workflows/set-version.yml , create an action that increases the version number. Also update the Readme file to point at that action.

**Comment by Jeehut** [comment]
This seems to be a duplicate of #100.

**Comment by Jeehut** [comment]
@winsmith I don't think we need this. Making a release is already as simple as bumping the version num, adding a tag in Git, and pushing. One has to add a tag anyways, and one needs a clear commit where the tag needs to be added to, which has always been a "Bump version num" commit which only does that. This seems pretty straightforward and the process doesn't hold any potential for forgetting to bump the version number. There are also not two places or so, it's exactly one place in code, so we have a single point of failure.

I think this can be closed without action.
